testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/DockerContainer.java

```diff
@@ -160,6 +161,12 @@ public DockerContainer withHealthCheck(Path healthCheckScript)
                 .withCreateContainerCmdModifier(command -> command.withHealthcheck(cmd));
     }
 
+    public DockerContainer temporary()
```

Comment:
Please add a javadoc to explain what it is the semantic of temporary container.

`setTemporary(true)`?